backend/src/packages/chaiNNer_pytorch/pytorch/processing/upscale_image.py

```diff
@@ -68,10 +68,13 @@ def estimate():
                 if options.use_fp16:
                     model_bytes = model_bytes // 2
                 mem_info: tuple[int, int] = torch.cuda.mem_get_info(device)  # type: ignore
-                free, _total = mem_info
+                _free, total = mem_info
+                # only use 75% of the total memory
+                total = int(total * 0.75)
```

Comment:
I feel like this is too conservative. Even with chainner, chroma, steam, and VSCode running, my GPU uses less than 10% (1GB) of my total VRAM. Plus, because of the 80% down there, we estimate with only 60% of total VRAM.

Honestly, I would drop the first `int(total * 0.75)` and change the 80% below to 75%. That should be conservative enough. If it consistently estimates too high for some users, they can manually set a memory budget.

Reply:
In my testing this was equivalent to what we had before, so maybe this needs to be re-thought a bit

Reply:
IMO: it's better to be too conservative with it than deal with the constant VRAM issues

Reply:
I would agree with that. If its not using enough VRAM, set a large tile size or set it to max. Auto should cause the fewest issues (imo)

Reply:
Alright. Then let's be more conservative for now. If we find that it's too much, we can raise it later.